src/server/vnparse.c

```diff
@@ -2486,6 +2481,147 @@ typedef struct resc_limit_entry {
 	resc_limit_t	*resc;
 } rl_entry;
 
+/* Reusing ATR_VFLAG_HOOK to mean that the resource key value pair was found
+ * in in execvnode
+ */
+#define IN_EXECVNODE_FLAG	ATR_VFLAG_HOOK
```

Comment:
I'm not sure that's the best idea. Should we define a new flag instead? If I was working in this file and hadn't read this comment, I'd probably be a little confused as to what is going on. Also, the comment contains "in in execvnode".

Reply:
Well I understand your concern, however the number of bits is limited, so its best not to waste them in such minor isolated operations and save them for major distributed operations in future. The new name define with apt comments used in the code serves as an informant to coder working in future. The extra `in` has been removed in commit https://github.com/PBSPro/pbspro/pull/1452/commits/8f309272ad00f7587c830cad07238cb40598e24a

Reply:
May be I was too conservative. So fixed it in commit https://github.com/PBSPro/pbspro/pull/1452/commits/b33b0c1672d0a8fc21ece2ce23438681e42db09f